doc/usage/configuration.rst

```diff
@@ -1804,10 +1804,28 @@ information.
    FreeFont family) is often easier than trying to make
    ``'pdflatex'`` work with the extra Unicode characters.
 
+   .. hint::
+
+      Contrarily to the situation with HTML output (and MathJaX math
+      rendering), Unicode literals from :rst:dir:`math` directive contents are
+      not handled natively by LaTeX without extra configuration:
+
+      - the only comprehensive solution is to use ``'xelatex'`` or
+        ``'lualatex'`` *and* to add ``r'\usepackage{unicode-math}'`` via the
+        :confval:`latex_elements` ``'preamble'`` key. By default ``α``
+        (U+03B1) input will be rendered as :math:`\alpha`. To keep it as
+        it was input, use ``r'\usepackage[math-style=literal]{unicode-math}'``.
+      - for ``'pdflatex'``, one way to obtain support for Unicode Greek (but
+        no further Unicode symbols) in math mark-up is to activate the support
+        for Greek in text (see discussion of the :confval:`latex_elements`
+        ``'fontenc'`` key) *and* to add ``r'\usepackage{alphabeta}'`` to
+        ``'preamble'``. An ``α`` (U+03B1) input will render as :math:`\alpha`.
+
```

Comment:
I did not comment on Japanese case: α is ok but for example 𝚹 is not in my brief testing.